RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -808,6 +848,10 @@ void MuonIdProducer::fillMuonId(edm::Event& iEvent, const edm::EventSetup& iSetu
    for ( const auto& chamber : info.chambers )
    {
      if (chamber.id.subdetId() == 3 && rpcHitHandle_.isValid()  ) continue; // Skip RPC chambers, they are taken care of below)
+
+     if (doME0_){
+       if  (chamber.id.subdetId() == MuonSubdetId::ME0 && me0HitHandle_.isValid()  ) continue; // Skip ME0 chambers, they are taken care of below
```

Comment:
If the doME0_ is disabled, there won't be any ME0s at all.  If it is enabled, it should still skip non-ME0 chambers.  Should the doME0_ be removed in any case?  My intention was to not check the handle if it wouldn't be there.